sde-cookbook/qtcreator/configure-qtcreator.py

```diff
@@ -113,8 +114,7 @@ def add_kit(self):
                         + " --Cxxtoolchain " + self.cxx_toolchain_id
                         + " --Ctoolchain " + self.cc_toolchain_id
                         + " --cmake " + self.cmake_id
-                        + " --qt " + self.qtversion_id
-                        + " --mkspec \"" + os.environ["QMAKESPEC"] + "\""
+                        # + qt_params if use_qt else ""
```

Comment:
Shouldn't this be used?

Reply:
God damn it' you're right, I'll fix!

Reply:
More interestingly, why did the tests pass?

Reply:
Sounds like dangerous false positives. Have you tried a clean build?

Reply:
I did actually, looking into it manually now.